Header header
uint32 node_count
Node[] node
uint32 edge_count
Edge[] edge

	
